app/static/js/slcNewProposal.js

```diff
@@ -305,3 +307,14 @@ function getCourseInstructors() {
                 .map((i,el) => $(el).data('username')).get()
                 .filter(val => (val))
 }
+
+
+const textareas = $(".textarea");
+const slcQuestionCharCounts = $(".slcQuestionCharCount");
```

Comment:
rename the variable as  slcQuestionCharCount

Reply:
done